test/tools/WebListener/README.md

```diff
@@ -59,6 +59,46 @@ Response when certificate is not provided in request:
 }
 ```
 
+## /Compression/Deflate/
+Returns the same results as the Get test with deflate compression.
+
+```powershell
+$uri = Get-WebListenerUrl -Test 'Compression' -TestValue 'Deflate'
+Invoke-RestMethod -Uri $uri -Headers $headers
+```
+
+```json
+{
+  "args": {},
+  "origin": "127.0.0.1",
+  "headers": {
+    "User-Agent": "Mozilla/5.0 (Windows NT; Windows NT 10.0; en-US) WindowsPowerShell/5.1.15063.608",
+    "Host": "localhost:8083"
+  },
+  "url": "http://localhost:8083/Compression/Deflate"
+}
+```
+
+## /Compression/Gzip/
+Returns the same results as the Get test with gzip compression.
+
+```powershell
+$uri = Get-WebListenerUrl -Test 'Compression' -TestValue 'Gzip'
+Invoke-RestMethod -Uri $uri -Headers $headers
+```
+
+```json
+{
+  "args": {},
+  "origin": "127.0.0.1",
+  "headers": {
+    "User-Agent": "Mozilla/5.0 (Windows NT; Windows NT 10.0; en-US) WindowsPowerShell/5.1.15063.608",
```

Comment:
Should this be PowerShell/6.0.0 instead of WindowsPowerShell?

Reply:
Fixed